src/main/java/dev/dkvl/womutils/beans/PlayerInfo.java

```diff
@@ -10,7 +10,7 @@
     String username;
     String displayName;
     String type;
-    String build;
+    PlayerBuild build;
     String country;
```

Comment:
If you want to, it would be great if you could add type and country enums as well. So we could show the flag and country instead of the country code in the future.